packages/SwingSet/src/natNum.js

```diff
@@ -0,0 +1,32 @@
+import { Nat } from '@agoric/nat';
+
+export const ZERO = 0n;
+export const ONE = 1n;
+
+export function toStr(num) {
+  return `${Nat(num)}`;
+}
+
+export function fromStr(str) {
+  return Nat(Number(str));
```

Comment:
This will overflow if there are too many digits
```suggestion
  return Nat(BigInt(str));
```

Reply:
Done